src/js/themes/base.d.ts

```diff
@@ -769,7 +778,6 @@ export interface ThemeType {
     extend?: ExtendType;
     icons?: {
       down?: any;
-      color?: ColorType;
```

Comment:
Are we not allowing the `color` theme style for an `icon`? how is that backward compatible?